src/components/theme/ContentMetadataTags/ContentMetadataTags.jsx

```diff
@@ -63,35 +54,16 @@ const ContentMetadataTags = (props) => {
 
   const contentImageInfo = getContentImageInfo();
 
-  const getTitle = () => {
-    const includeSiteTitle =
-      config?.settings?.siteTitleFormat?.includeSiteTitle || false;
-    const titleAndSiteTitleSeparator =
-      config?.settings?.titleAndSiteTitleSeparator || '-';
-    const navRootTitle = navroot?.title;
-    const siteRootTitle = site?.['plone.site_title'];
-    const titlePart = navRootTitle || siteRootTitle;
-
-    if (includeSiteTitle && titlePart && titlePart !== title) {
-      return seo_title || `${title} ${titleAndSiteTitleSeparator} ${titlePart}`;
-    } else {
-      return seo_title || title;
-    }
-  };
-
```

Comment:
Why have you removed this?

This information isn't coming form the Header (https://github.com/plone/volto/blob/7b85e97c5e7c5046f1fce432bbbfd3f6842bd15f/src/components/theme/View/View.jsx#L238) so suddenly this feature has dissapeared